src/quo2/components/common/not_implemented/style.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo2.components.common.not-implemented.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn text
+  [blur?]
+  {:border-color :red
+   :border-width 1
+   :color        (if
+                   blur?
+                   colors/white
+                   (colors/theme-colors colors/neutral-100
```

Comment:
I know this component is only a temporary piece but eventually we want to make the 3rd param of `colors/theme-colors` mandatory- could you please use the agreed approach to theming mechanism 🙏

Reply:
Done

Reply:
Also please put `blur?` in the same line as `if` in line 8